types/sinon/ts3.1/sinon-tests.ts

```diff
@@ -424,10 +423,11 @@ function testSpy() {
 }
 
 function testStub() {
+    const promiseLike = { then: () => promiseLike };
```

Comment:
Eeeh, why not `const promiseLIke: PromiseLike = Promise.resolve()`
This seems like a weird construct.

Reply:
yup i would go with that too. i was leaning more towards adding a promise-like interface last time but you are right that we can just add a `Promise.resolve`.

Reply:
agreed, looks cleaner this way

fixed, thanks